docs/design/autoware-interfaces/components/sensing.md

```diff
@@ -0,0 +1,46 @@
+# Sensing
+
+![Node diagram](images/Sensing-Bus-ODD-Architecture.drawio.svg)
+
+## Inputs
+
+### The sensing module contains the driver for the sensor, so there is no input
+
+## Output
+
+### Camera image
+
+Image data from camera. Used by the Perception.
+
+- [sensor_msgs/Image](https://docs.ros.org/en/noetic/api/sensor_msgs/html/msg/Image.html)
+
+### Lidar original pointcloud
+
+Pointcloud after preprocession. Used by the Perception.
+
+- [sensor_msgs/PointCloud2](https://docs.ros.org/en/noetic/api/sensor_msgs/html/msg/PointCloud2.html)
+
+### Lidar filtered pointcloud
+
+Pointcloud after preprocession. Used by the Perception.
+
+- [sensor_msgs/PointCloud2](https://docs.ros.org/en/noetic/api/sensor_msgs/html/msg/PointCloud2.html)
+
+### Radar pointcloud
+
+Pointcloud frome radar. Used by the Perception.
```

Comment:
Please check the spell (You can find it from the "Files changed" tab)
![image](https://user-images.githubusercontent.com/44218668/227883077-81593c35-5012-4d14-bf89-a708078f7c73.png)